uw-frame-components/portal/notifications/partials/notifications-full.html

```diff
@@ -1,41 +1,99 @@
-              </div>
-        		</li>
-        		<li class="notification-item read" ng-if="isEmpty || (countWithDismissed() === 0 && mode === 'new') || (mode === 'old' && dismissedCount() === 0)">You have no <span ng-show='dismissedCount() === 0 && countWithDismissed() !== 0'>dismissed</span> notifications at this time.</li>
-        	</ul>
-		</div>
-	</div>
+    <md-tabs md-dynamic-height md-border-bottom>
+      <md-tab aria-label="click to see new notifications" label="New">
+        <md-content>
+          <md-list class="notifications-list" flex>
+            <md-list-item class="secondary-button-padding notification-item"
+                          ng-class="{ priority: notification.priority }"
+                          ng-repeat="notification in notifications | orderBy:['-priority', '-id']"
+                          ng-if="!isDismissed(notification)"
+                          layout="row"
+                          layout-align="start center">
+              <!-- NOTIFICATION TEXT AND ACTION BUTTONS -->
+              <a class="md-list-item-text"
+                 ng-href="{{ notification.actionURL }}"
+                 target="_blank"
+                 aria-label="{{ notification.actionAlt }}"
+                 layout="row"
+                 layout-align="start center"
+                 layout-xs="column"
+                 layout-align-xs="center start">
+                <span><i class="fa fa-exclamation-triangle" ng-if="notification.priority"></i> {{ notification.title }}</span>
+                <div ng-if="notification.actionButtons && notification.actionButtons.length > 0" class="notification-buttons">
+                  <md-button class="md-raised"
+                             ng-href="{{button.url}}"
+                             ng-repeat="button in notification.actionButtons track by button.label"
+                             ng-class="{'md-primary' : $index === 0, 'md-accent' : $index > notification.buttonText.length / 2 - 1}"
+                             ng-click="button.action"
+                             target="{{button.target}}"
+                             aria-label="{{button.label}}">
+                    {{button.label}}
+                  </md-button>
+                </div>
+              </a>
+              <!-- DISMISS BUTTON -->
+              <md-button class="md-icon-button md-secondary"
+                         ng-click="dismiss(notification)"
+                         ng-hide="!notification.dismissable || isDismissed(notification)"
+                         aria-label="dismiss this notification">
+                <i class='fa fa-times'></i>
+              </md-button>
+            </md-list-item>
+            <!-- IF NO NOTIFICATIONS -->
+            <md-list-item class="notification-item read"
+                          ng-if="isEmpty || (countWithDismissed() === 0)">You have no notifications at this time.
```

Comment:
Maybe emphasize that have no *new* notifications, since might have dismissed notifications? And maybe lose the "at this time." for brevity? Yielding

"You have no new notifications."